mmaction/datasets/pipelines/loading.py

```diff
@@ -910,6 +910,122 @@ def __call__(self, results):
 
 
 @PIPELINES.register_module()
+class AudioDecodeInit(object):
+    """Using librosa to initialize the audio_reader.
+
+    Args:
+        io_backend (str): io backend where frames are store.
+            Default: 'disk'.
+        sample_rate (int): Audio sampling times per second. Default: 16000.
+
+    Required keys are "audio_path", added or modified keys are "length",
+    "sample_rate", "audios".
+    """
+
+    def __init__(self, io_backend='disk', sample_rate=16000, **kwargs):
+        self.io_backend = io_backend
+        self.sample_rate = sample_rate
+
+        self.kwargs = kwargs
+        self.file_client = None
+
+    def __call__(self, results):
+        """Perform the librosa initiating.
```

Comment:
`initiating` -> `initialization`

Reply:
BTW some classes like PyAVInit are using `initiation`, while `initialization` is more precise. I have revised them in this PR.

Reply:
Agree